package.json

```diff
@@ -119,10 +118,11 @@
     "semantic-release": "lerna exec --concurrency 1 -- semantic-release -e semantic-release-monorepo -e $(pwd)/release.config.mjs",
     "test": "pnpm lint:js && pnpm lint:styles && pnpm check:i18n-en-fr && pnpm typescript && pnpm unit && pnpm test-storybook && pnpm a11y-test",
     "typescript": "lerna run tsc",
-    "unit": "jest --testPathIgnorePatterns a11y .d.ts",
+    "unit": "jest --testPathIgnorePatterns esm lib a11y .d.ts",
```

Comment:
Why are tests showing up in esm and lib folders? We do not want to be packaging these

Reply:
oops that was a mistake

Reply:
oh i must have reverted the change oops